meson-cross-mipsel.ini

```diff
@@ -0,0 +1,15 @@
+[constants]
+arch = 'mipsel'
+
+[binaries]
+c = arch + '-openwrt-linux-gcc'
```

Comment:
where do we get that cross-compiler from? Can you add it to our CI so we don't regress?